.travis.yml

```diff
@@ -17,10 +17,11 @@ before_install:
     - ./scripts/install-dependencies.rb
 
 script:
-    - ./scripts/install-robolectric.sh
+    - ./gradlew clean assemble publishToMavenLocal compileTest --info --stacktrace
+    - ./gradlew --continue test --info --stacktrace
 
 after_success:
-    - ./scripts/deploy-snapshot.sh
+#    - ./scripts/deploy-snapshot.sh
```

Comment:
Ah right, I didn't want to trash our snapshots while this was thrashing; I'll turn this back on. For now it'll push to `3.2-gradle-SNAPSHOT`; we should switch that back once it feels solid.